src/ngMock/angular-mocks.js

```diff
@@ -2072,13 +2072,13 @@ function MockXhr() {
     var header = this.$$respHeaders[name];
     if (header) return header;
 
-    name = angular.lowercase(name);
+    name = angular.$$lowercase(name);
```

Comment:
I just noticed that angular.lowercase / uppercase is overwritten with a manual version that handles Turkish `i`: https://github.com/angular/angular.js/blob/19bc52127f8393ffccac563b21267adb75c19275/src/Angular.js#L210-L232

So in core, we use a "safe" version. But we don't want anyone else to use it? I guess we don't want anyone to use the special version when usually the prototype functions are enough.